docs/api_v2/api_v2_reference.apib

```diff
@@ -2918,238 +3123,413 @@ The user account is identified by the provided authentication token.
 
 ## Build (object)
 - id:               `HG7ML7M8z78YcAPEB`        (string, required)
-- actId:            `johndoe~my-actor`         (string, required)
+- actId:            `janedoe~my-actor`         (string, required)
 - userId:           `klmdEpoiojmdEMlk3`        (string, required)
-- startedAt:        `2015-11-30T07:34:24.202Z` (string, required)
-- finishedAt:       `2015-12-12T09:30:12.202Z` (string, nullable)
+- startedAt:        `2019-11-30T07:34:24.202Z` (string, required)
+- finishedAt:       `2019-12-12T09:30:12.202Z` (string, nullable)
 - status:           `SUCCEEDED`                (string, required)
 - meta                                         (object, required)
-    - source:        `WEB`                     (string, required)
+    - origin:        `WEB`                     (string, required)
     - clientIp:      `172.234.12.34`           (string, required)
     - userAgent:     `Mozilla/5.0 (iPad)`      (string, required)
 - stats                                        (object, nullable)
     - workersUsed:   1                         (number, required)
-    - durationMillis 1000                      (number, required)
-- buildNumber:       `0.0.1`                   (string, required)
-- readme:            `# Magic actor\nThis actor is magic.` (string, nullable)
+    - durationMillis: 1000                     (number, required)
+    - runTimeSecs:   45.718                    (number, required)
+    - readyTimeSecs: 0.035                     (number, required)
+    - initTotalSecs: 0.139                     (number, required)
+    - emfileError                              (object, nullable)
+    - computeUnits:  0.012699444444444444      (number, required)
+- options                                      (object, nullable)
+    - useCache:      false                     (boolean, nullable)
+    - betaPackages:  false                     (boolean, nullable)
+    - memoryMbytes:  1024                      (number, nullable)
+    - diskMbytes:    2048                      (number, nullable)
```

Comment:
`diskMbytes` are quite useful for users to know but I am afraid that having this in options could confuse them as they might be trying to override it? @jancurn

Reply:
well, when they are hackers enough and try to override it, they will see it does nothing... imho we can keep it